src/components/Semester/Semester.vue

```diff
@@ -455,15 +470,28 @@ export default defineComponent({
       this.openConfirmationModal(`Added ${course.code} to ${this.season} ${this.year}`);
     },
     deleteCourseWithoutModal(uniqueID: number) {
-      deleteCourseFromSemester(this.year, this.season, uniqueID, this.$gtag);
+      deleteCourseFromSemester(
+        store.state.currentPlan,
```

Comment:
Why do we need to pass the current plan as an argument?

Reply:
I guess there's no case where you aren't deleting from the current plan currently but it was just the way I wrote the firestore functions to modify the plan that is passed in.